.gitignore

```diff
@@ -16,9 +16,6 @@ __pycache__/
 *.log
 *.ipynb_checkpoints
 *.xlsx#
-!WF1_Template.xlsx
-!WF1_DevTemplate.xlsx
-!WF1_Alloy_Template.xlsx
-!WF3_template.xlsx
-!WF3_DevTemplate.xlsx
-!MIT_PVLab_Template.xlsx
+!fixed_wf3_template.xlsx
```

Comment:
Does fixed imply... not broken? 
Either way, we should probably rename before merging. 

@ipendlet is there anything different about the *logic* in `fixed_wf3_template.xlsx` when compared to previous WF3 templates, or is the only difference in content?